pom.xml

```diff
@@ -409,7 +403,7 @@
             </modules>
         </profile>
         <profile>
-            <id>central-deploy</id>
+            <id>coverage</id>
```

Comment:
Why have we configured jacoco twice? Why not a single profile for code coverage?